equinox-shipment/Domain/FinalizationTransaction.fs

```diff
@@ -0,0 +1,90 @@
+module FinalizationTransaction
+
+module Events =
+
+    let [<Literal>] CategoryId = "FinalizationTransaction"
+
+    let (|ForClientId|) (clientId: string) = FsCodec.StreamName.create CategoryId clientId
+
+    type Event =
+        | FinalizationRequested of containerId: string * shipmentIds: string[]
+
+        | AssignmentCompleted   of containerId: string * shipmentIds: string[]
+        | FinalizationCompleted of containerId: string
+
+        | RevertRequested       of containerId: string * shipmentIds: string[]
```

Comment:
RevertRequired ?
containerId should only be in FinalizationRequested event
If AssignmentCompleted is not bringing new info to the table it should also not have shipmentIds

Reply:
I've called it `RevertRequested` to indicated that a revert will be in process.

Reply:
closing, will review updated form